### ISSUE TITLE
[RFE]: 'get <jail> banip --with-time' output IP address formatting improvement 

### ISSUE THREAD

**Issue by findurance** [fail2ban/fail2ban#3394]
#### Feature request type
Improve output formatting for readability

#### Description
The IP-address output field width could be set to fixed 15 charater width, according to max. IP address length (12 numbers + 3 dots). Currently the output columns have varying offsets:

```
> sudo fail2ban-client get sshd banip --with-time
197.221.130.254         2022-10-29 21:59:41 + 3600 = 2022-10-29 22:59:41
212.179.12.206  2022-10-29 22:01:02 + 3600 = 2022-10-29 23:01:02
159.65.103.250  2022-10-29 22:30:53 + 3600 = 2022-10-29 23:30:53
124.137.205.59  2022-10-29 22:31:15 + 3600 = 2022-10-29 23:31:15
```

**Comment by sebres** [comment]
> The IP-address output field width could be set to fixed 15 charater width, according to max

No. It could be not. IIRC largest IPv6 is 39 chars (8 * 4 + 7 = 39, 8 times of 4 hex chars with 7 colons between).

What we can do would be to add some spaces before the tab char, to align it more sane... Just it'd be a bit incompatible to current variant (maybe unexpected for some parsers).

**Comment by findurance** [comment]
> No. It could be not. IIRC largest IPv6 is 39 chars (8 * 4 + 7 = 39, 8 times of 4 hex chars with 7 colons between).

Well sure, for IPv6 the field width would be the maximum of 39 and for IPv4 15. But I'm not too sure if tab is the right charater for indentation here anyways. I'd at least consider the fixed width address field..
